pillar/nginx/chalkradio_redirect.sls

```diff
@@ -0,0 +1,34 @@
+nginx:
+    servers:
+    managed:
+      {% for tld in tlds %}
```

Comment:
This for loop can go now that we've got all of the domains in the server_name field

Reply:
The templated `tld` in the next line will also need to be removed